Cargo.toml

```diff
@@ -98,7 +103,7 @@ rustls-tls-native-roots = [
 
 [dependencies]
 clickhouse-derive = { version = "0.2.0", path = "derive" }
-
+clickhouse-types = { version = "*", path = "types" }
```

Comment:
Later versions of `clickhouse-types` can introduce breaking changes, so you should use an actual version here.

cargo-release can detect changes on its own, but we need to configure it to work with the workspace